### PR TITLE
Upgrade to v1.39.0 of the AzureRM Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: resources.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ const (
 	azureAD                  = "AD"                  // Active Directory (AAD)
 	azureAnalysisServices    = "AnalysisServices"    // Analysis Services
 	azureAPIManagement       = "ApiManagement"       // API Management
+	azureAppConfiguration    = "AppConfiguration"    // AppConfiguration
```
Comment (mikhailshilkov) [inline resources.go]: Nit to match the product name
```suggestion
	azureAppConfiguration    = "AppConfiguration"    // App Configuration
```

File: resources.go [hunk 2]
```diff
@@ -838,6 +839,15 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_mysql_firewall_rule":        {Tok: azureResource(azureMySQL, "FirewallRule")},
 			"azurerm_mysql_server":               {Tok: azureResource(azureMySQL, "Server")},
 			"azurerm_mysql_virtual_network_rule": {Tok: azureResource(azureMySQL, "VirtualNetworkRule")},
+			"azurerm_mssql_database_vulnerability_assessment_rule_baseline": {
+				Tok: azureResource(azureMySQL, "DatabaseVulnerabilityAssessmentRuleBaseline"),
```
Comment (mikhailshilkov) [inline resources.go]: This is not MySQL, this is MS SQL (same for resources below)

File: resources.go [hunk 3]
```diff
@@ -1194,6 +1204,7 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_bot_channel_email":         {Tok: azureResource(azureBot, "ChannelEmail")},
 			"azurerm_bot_channel_slack":         {Tok: azureResource(azureBot, "ChannelSlack")},
 			"azurerm_bot_web_app":               {Tok: azureResource(azureBot, "WebApp")},
+			"azurerm_bot_channel_ms_teams":      {Tok: azureResource(azureBot, "ChannelMsTeams")},
```
Comment (mikhailshilkov) [inline resources.go]: Shouldn't `MS` be capital? At least, it should in C# naming conventions. Alternatively, I'm fine with the name "ChannelTeams".

File: resources.go [hunk 4]
```diff
@@ -1210,6 +1221,9 @@ func Provider() tfbridge.ProviderInfo {
 			// NetApp
 			"azurerm_netapp_account": {Tok: azureResource(azureNetapp, "Account")},
 			"azurerm_netapp_pool":    {Tok: azureResource(azureNetapp, "Pool")},
+
+			//AppConfiguration
+			"azurerm_app_configuration": {Tok: azureResource(azureAppConfiguration, "ConfigurationStore")},
```
Comment (mikhailshilkov) [inline resources.go]: I'm happy with `ConfigurationStore`. `Store` could read more natural in TypeScript, but is more explicit in namespace-less .NET code.

File: resources.go (hunk 3 above)
Comment (stack72) [reply resources.go]: I Like `ChannelTeams` better :) Moving to that!

File: resources.go (hunk 4 above)
Comment (stack72) [reply resources.go]: 👍

File: resources.go (hunk 2 above)
Comment (stack72) [reply resources.go]: Great spot!!!

**Pull request merged**